pkg/apis/hive/v1/hiveconfig_types.go

```diff
@@ -172,14 +170,10 @@ type ManageDNSAzureConfig struct {
 	// managed domains for this cluster.
 	// listed in the parent ManageDNSConfig object.
 	// Secret should have a key named 'osServicePrincipal.json'
-	// +optional
 	CredentialsSecretRef corev1.LocalObjectReference `json:"credentialsSecretRef,omitempty"`
 
-	// +optional
-	Region string `json:"region,omitempty"`
-
-	// +optional
-	BaseDomainResourceGroupName string `json:"baseDomainResourceGroupName,omitempty"`
+	// ResourceGroupName specifies where the DNS Zone should be created
```

Comment:
This description is not quite right since Hive does not create any DNS zones in this resource group.
```suggestion
	// ResourceGroupName specifies the Azure resource group containing the DNS zones for the domains being managed.
```

Reply:
This makes more sense. Thanks. Updated